BraveRewardsUI/Pending Contributions/PendingContributionDetailController.swift

```diff
@@ -0,0 +1,116 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveRewards
+
+final class PendingContributionDetailController: UIViewController {
+  private let state: RewardsState
+  private let contribution: PendingContributionInfo
+  
+  init(state: RewardsState, contribution: PendingContributionInfo) {
+    self.state = state
+    self.contribution = contribution
+    super.init(nibName: nil, bundle: nil)
+  }
+  
+  @available(*, unavailable)
+  required init(coder: NSCoder) {
+    fatalError()
+  }
+  
+  private var contentView: SettingsTableView {
+    return view as! SettingsTableView // swiftlint:disable:this force_cast
+  }
+  
+  override func loadView() {
+    view = SettingsTableView()
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    contentView.tableView.delegate = self
+    contentView.tableView.dataSource = self
+    
+    contentView.tableView.register(Value1TableViewCell.self)
+    contentView.tableView.register(TableViewCell.self)
+    
+    title = contribution.name
+  }
+  
+  private let dateFormatter = DateFormatter().then {
+    $0.dateStyle = .short
+  }
+}
+
+extension PendingContributionDetailController: UITableViewDelegate {
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    if indexPath.section != Section.remove.rawValue { return }
+    state.ledger.removePendingContribution(contribution) { _ in
+      self.navigationController?.popViewController(animated: true)
+    }
+  }
+}
+
+extension PendingContributionDetailController: UITableViewDataSource {
+  enum Section: Int, CaseIterable {
+    case details
+    case remove
+  }
+  enum DetailsRow: Int, CaseIterable {
+    case type
+    case amount
+    case pendingUntil
+  }
+  
+  func numberOfSections(in tableView: UITableView) -> Int {
+    return Section.allCases.count
```

Comment:
same thing with single expression returns, can leave if you like it that way better